api/types.proto

```diff
@@ -696,3 +696,9 @@ message ManagerStatus {
 	// Reachability specifies whether this node is reachable.
 	RaftMemberStatus.Reachability reachability = 4;
 }
+
+message Device {
+	string source = 1;
+	string target = 2;
+	string permissions = 3;
```

Comment:
Are you referring to these structures?
https://github.com/opencontainers/runtime-spec/blob/c9c25740dfb920a32929eda1ed45785011b5a5fc/specs-go/config.go#L324-L354

For this part, I did something similar to engine-api:
https://github.com/docker/engine-api/blob/6b57ada6604da94fedc0c96e51f6ec74a10b1acd/types/container/host_config.go#L182-L187

What are you suggesting here?